magento/pwa-studio
# Adding host headers to the GQL calls from buildpack.

## Description

Adding host headers to the GQL calls from buildpack to resolve those requests.

There is another CICD PR we need to merge along with this: https://github.com/magento-commerce/pwa-studio-cicd/pull/78

## Related Issue

Closes PWA-1987

## Acceptance

Scaffolding job should not fail in the CI.

### Verification Stakeholders

@sharkySharks 

## Verification Steps

1. Check the job once this is merged for the job to consider using new code.
Meanwhile, if you need to test this code, re-run this [new temporary job that is a replica of the original scaffolding job](https://console.aws.amazon.com/codesuite/codebuild/913454440186/projects/pwa-scaffold-build-test/history?region=us-east-1&builds-meta=eyJmIjp7InRleHQiOiIifSwicyI6e30sIm4iOjIwLCJpIjowfQ#).

## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
